lib/main.dart

```diff
@@ -3,13 +3,22 @@ import 'package:flutter/material.dart';
 void main() => runApp(MyApp());
 
 class MyApp extends StatelessWidget {
+
+  hexColor (String colorhexcode) {
+    String colornew = '0xff' + colorhexcode;
+    colornew = colornew.replaceAll('#', '');
+    int colorint = int.parse(colornew);
+    return colorint;
+  }
+
```

Comment:
Transfer this to a different file. You can just do a simple file with a function or make use of the extension function capability of Dart.

Reference: https://stackoverflow.com/questions/48836636/how-to-use-functions-of-another-file-in-dart-flutter
https://dart.dev/guides/language/extension-methods

Reply:
Done